### DISCUSSION TITLE
AdManager REST API Missing Services from SOAP

### DISCUSSION THREAD

**Discussion by TLadd** [googleapis/googleapis#912] [Q&A]
I'm a present consumer of the ad manager SOAP API, and I was happy to see protos added about 3 months ago for a grpc/rest API https://github.com/googleapis/googleapis/tree/master/google/ads/admanager/v1 that [looks like it's in beta](https://developers.google.com/ad-manager/api/beta/getting-started).

However, I notice a couple of services that I consume that don't have a matching service in the REST API. Examples:

- https://developers.google.com/ad-manager/api/reference/v202405/CreativeSetService
- https://developers.google.com/ad-manager/api/reference/v202405/InventoryService
- https://developers.google.com/ad-manager/api/reference/v202405/LineItemCreativeAssociationService
- https://developers.google.com/ad-manager/api/reference/v202405/PublisherQueryLanguageService
- https://developers.google.com/ad-manager/api/reference/v202405/SiteService

I'm wondering if the REST API will eventually reach parity or if these services' functionality is moving elsewhere.

**Comment by jfrconley** [comment]
So.... Is this just not happening? A year and half later and the API is still missing most of this functionality and many services only offer read only access. Why bother even having it?

**Comment by kovashikawa** [comment]
do we have any updates on this? @codyoss , I noticed you approved some code reviews recently. 

could you please review this and address it internally?
